src/main/java/com/assertthat/selenium_shutterbug/utils/web/Browser.java

```diff
@@ -109,8 +109,8 @@ public BufferedImage takeScreenshot() {
      * @return BufferedImage resulting image
      */
     public BufferedImage takeScreenshotEntirePage() {
-        if (driver instanceof EventFiringWebDriver) {
```

Comment:
I would suggest adding something like 

``` 
   private WebDriver unwrapDriver(String name){
        try {
            if(Class.forName(name).isInstance(driver)){
                return ((WrapsDriver) driver).getWrappedDriver();
            }
        } catch (ClassNotFoundException e) {
            ;
        }
        return driver;
    }
```

So that then in takeScreeshotEntirePage() we can be backward compatible with earlier selenium versions e.g. 

```       
 driver = unwrapDriver("org.openqa.selenium.WrapsDriver");
 driver = unwrapDriver("org.openqa.selenium.internal.WrapsDriver");
```

Reply:
Since you have bumped the Selenium version in the meantime, I would rather rebase my branch and switch to `org.openqa.selenium.WrapsDriver`, which is implemented by `org.openqa.selenium.internal.WrapsDriver`, so this would support both. (See subinterfaces: https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WrapsDriver.html).